hcsshim.go

```diff
@@ -119,3 +165,35 @@ func convertAndFreeCoTaskMemString(buffer *uint16) string {
 	coTaskMemFree(unsafe.Pointer(buffer))
 	return str
 }
+
+func convertAndFreeCoTaskMemBytes(buffer *uint16) ([]byte, error) {
```

Comment:
Oh, this is much more complicated than I anticipated. Sorry! -- I think you should go back to the old approach.